xml/cap_depl_install_caasp.xml

```diff
@@ -410,12 +410,18 @@ pvc/test-sc-persistent   Bound     pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c   1G
    volume claims when you're finished:
   </para>
 
-<screen>&prompt.user;kubectl delete pv/pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c
-persistentvolume "pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c" deleted
-
-&prompt.user;kubectl delete pvc/test-sc-persistent
+<screen>&prompt.user;kubectl delete pvc/test-sc-persistent
 persistentvolumeclaim "test-sc-persistent" deleted</screen>
 
+  <tip>
+    <title>Deleting persistent volume</title>
+    <para>
+      Usually you have to delete the persistent volume manually but
+      in this case the persistent volume has <literal>reclaimPolicy: Delete</literal>
+      set and thus it is done automatically.
```

Comment:
Might need to reword the start of the sentence (the "Usually you ... manually" portion) slightly. A reclaim policy is either `Retain` or `Delete` (with `Delete` being the default) so the current phrasing at is not entire accurate.

https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming

Reply:
Changed the wording, let me know if it's better this way.